src/Tasks/Common/ConflictResolution/ConflictItem.cs

```diff
@@ -103,7 +108,7 @@ public string FileName
             {
                 if (_fileName == null)
                 {
-                    _fileName = OriginalItem == null ? String.Empty : OriginalItem.GetMetadata(MetadataNames.FileName) + OriginalItem.GetMetadata(MetadataNames.Extension);
+                    _fileName = OriginalItem == null ? String.Empty : Path.GetFileName(OriginalItem.ItemSpec);
```

Comment:
FYI - this change is unrelated to the package override work, but I saw roughly a 200ms gain in OrchardCore by making this change.